webpack.config.js

```diff
@@ -492,10 +492,24 @@ if (
 //
 // Various sentry pages still rely on django to serve html views.
 if (IS_UI_DEV_ONLY) {
+  // Try and load certificates from mkcert if available
+  //
+  // $ mkcert \
+  //   -key-file config/webpackHttps/localhost-key.pem \
+  //   -cert-file config/webpackHttps/localhost.pem \
+  //   localhost
```

Comment:
Can we add a yarn script for this? can we also just keep in in `config` root?

Reply:
sure to both 👍

Reply:
done